apps/src/gamelab/GameLab.js

```diff
@@ -88,6 +128,13 @@ GameLab.prototype.init = function (config) {
   this.skin = config.skin;
   this.level = config.level;
 
+  // Pre-populate asset list
+  assetsApi.ajax('GET', '', function (xhr) {
```

Comment:
This code is common with applab.js as well.  I know it's smaller, but it seems like it belongs in some common helper named `prepopulateAssetList`.